js/init.js

```diff
@@ -0,0 +1,8 @@
+(function($){
```

Comment:
why...

also, if we do have a js bucket, we should refactor the knapsack formula into it from cabinets.html